### ISSUE TITLE
Best way to avoid login screen for dashboard

### ISSUE THREAD

**Issue by bartbutenaers** [bartbutenaers/Node-RED-Tailscale-Tutorial#2]
In this [section](https://github.com/bartbutenaers/Node-RED-Tailscale-Tutorial/blob/main/docs/extend_reverse_proxy.md#access-the-node-red-dashboard) of the tutorial it is described how to avoid a login screen for the dashboard, while the flow editor is still behind a login screen.

Not sure if that way of working has any security related issues.  For example does this really only allow access to the dashboard user interface, or do I open some crucial endpoints this way?

**Comment by TotallyInformation** [comment]
Ah, I missed this bit when looking before. You have the editor on port 443 and the Dashboard on 8443. I would always do it the other way around. Your most public endpoints should be on the default port (443) and the more private ones (e.g. the editor) on a "random" high port. This is generally a little more secure since people cannot stumble on your editor as easily.

Oh, I can see from the next part of that both are on 443. Slightly confusing from the first section and I had to read it a couple of times.

Personally, I would always put the editor somewhere less easy to find - and preferably not make it accessible to the Internet or your Tailnet at all if you don't really have to.

As for removing the editor login - does anyone but you need access to that? If not, simply don't make it available! If you do make it available either use your proxy to add modern authentication (not sure if Tailscale has that capability?) or retain the login since if it is just you, your browser(s) will likely retain the access token anyway.

In my view, authentication is best done by a tool that knows how to do it properly and minimises opportunities for messing up. If using Cloudflare Zero Trust or NGROK for example, far better to secure an endpoint using their security rather than trying to get something generic like Node-RED to do it.

**Comment by bartbutenaers** [comment]
@TotallyInformation 
Until last week I had indeed used 443 for the funnel, and 8443 for the web interfaces.  Which ike you say is not optimal. So I switched them last week and updated the tutorial.
Could you please  pass me a screenshot where I forgot to update it.  Will update it as soon as I get home, because that will be very confusing for the readers...

**Comment by TotallyInformation** [comment]
![image](https://github.com/user-attachments/assets/3457f82b-b03f-4f9e-8e67-c92907bf8f8d)

**Comment by bartbutenaers** [comment]
Morning @TotallyInformation,

> Slightly confusing from the first section and I had to read it a couple of times.

Sorry but I really can't find where there is a mismatch.  I interpret it as that somewhere I say that dashboard is running at port 8443 (instead of 443)?  You will have to assist me finding it I am afraid...

> Personally, I would always put the editor somewhere less easy to find

Yes no arguments against that.  Of course I need to walk the thin line between security and user-friendlyness.  Therefore I think it is ok that I show people how to make the editor available in their tailnet, but also give thems some hints about the risks and give them some tips.  So I have added this:

+ Update 1:

   ![image](https://github.com/user-attachments/assets/34f07629-edbc-471f-a5c9-873fdd6e60a7)

+ Update 2:

   ![image](https://github.com/user-attachments/assets/e6cbc1d6-bfc4-4765-8610-55df4b90c155)

+ Update 3:

   ![image](https://github.com/user-attachments/assets/f7ac8707-db82-4f04-835b-0c73fe0774f8)

Then the choice is up to the user, but at least they are aware of it...

**Comment by bartbutenaers** [comment]
> In my view, authentication is best done by a tool that knows how to do it properly 

True.  I have added a comment about that and created a feature request in the Tailscale repo:

![image](https://github.com/user-attachments/assets/5bc28cb6-4258-4c36-8328-039b196831b0)
